peps/pep-0396.rst

```diff
@@ -1,15 +1,21 @@
 PEP: 396
 Title: Module Version Numbers
-Version: $Revision: 65628 $
-Last-Modified: $Date: 2008-08-10 09:59:20 -0400 (Sun, 10 Aug 2008) $
 Author: Barry Warsaw <barry@python.org>
-Status: Rejected
+Status: Withdrawn
 Type: Informational
 Topic: Packaging
-Content-Type: text/x-rst
 Created: 16-Mar-2011
 Post-History: 05-Apr-2011
 
+.. withdrawn::
+
+   Refer to :ref:`packaging:runtime-version-access` and
```

Comment:
BTW, that link says:

> see the rejected proposal in [PEP 396](https://peps.python.org/pep-0396/)

so I think it also needs to be changed to say "withdrawn".

Reply:
Yeah, I noticed that too. I'm a packaging user guide editor, so I'll fix it after merging this status update.